src/helpers/validationUtils.ts

```diff
@@ -0,0 +1,24 @@
+import { COUNTRIES } from './constants';
+
+/**
+ * Returns `true` when the provided country code is a valid
+ * and supported code, otherwise returns `false`.
+ * @param country | 2 digit country code
+ * @returns {boolean}
+ */
+const validateCountryCode = (country: string): boolean => {
+    if (country.length !== 2) return false;
+    return !!COUNTRIES.find((c) => c.abbrev === country);
```

Comment:
Why not `return COUNTRIES.find((c) => c.abbrev === country)` in this case? `.find()` returns `undefined` if it is unsatisfied.

Reply:
Because if it is satisfied it returns what it found. In this case `{country: 'United States', abbrev: 'US'}` but I just want a boolean.